src/main/java/frc/robot/commands/AutoBalance.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.PIDCommand;
+import frc.robot.subsystems.DriveTrainSubsystem;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+public class AutoBalance extends PIDCommand {
+  /** Creates a new AutoBalance. */
+  private DriveTrainSubsystem m_driveTrain; 
+
+
+  public AutoBalance(DriveTrainSubsystem driveTrain) {
+    super(
+        // The controller that the command will use
+        new PIDController(0, 0, 0), // Need to insert proper pid values here, waiting until testing
```

Comment:
do you want to make these values pull and update from the sendable chooser / from the dashboard somehow?

Reply:
Yes, will probably make them a smartdashboard value once we can get to testing the actual values